server.js

```diff
@@ -23,7 +38,7 @@ app.get('/books/:id',function (req,res) {
 
 app.post('/books',function(req,res){
 var bookdata=req.body;
-var newBook = new Book(bookdata);
+var newBook=new Book(bookdata);
```

Comment:
`const newBook = new Book( req.body );`

you don't need `bookdata`

Reply:
okay